### DISCUSSION TITLE
Advanced Hosting issues - Anyone running it currently?

### DISCUSSION THREAD

**Discussion by hutchred** [bullhorn/career-portal#518] [Q&A]
Hey Everyone,

Has anyone recently implemented the Advanced Hosting version of the OSCP in recent months? We've been working on this for a while but are running into issues. 

We've followed the following steps

**Building
In order to build Bullhorn Career Portal, ensure that you have [Git](http://git-scm.com/downloads), [Node.js](http://nodejs.org/), and [Angular CLI](https://angular.io/guide/setup-local#step-1-install-the-angular-cli) installed.
> Clone a copy of the repo:
> git clone https://github.com/bullhorn/career-portal.git
> Change to the Career Portal directory:
> cd career-portal
> Install build tools and dev dependencies:
> npm install
> Use one of the following to build and test:
> npm run serve           # Launch a local version of the career portal (frontend only).
> npm start               # Launch a local server with server side rendering portal.
> npm run build:static    # Build an optimized version of Career Portal in `/dist`
> npm run build          # Build a package for use with server side rendering**

Everything points that the latest version of the portal is 3.5.0 but when I DL the source code, we're seeing 3.3 at best. That's also all I see in the logs during the npm build process

 

We've gotten the standard hosting of the site to run just fine but we're missing something. The portal sits on an Azure server running IIS. Everything I've read shows it is supported. 

Are we missing something? Does anyone have any recommendations that I can look at?

**Comment by pbearne** [comment]
If you don't get this working Check out matador Jobs as our solution solves all the SEO issues (and more) that the Advance hosting is needed for
